extensions/amp-render/1.0/amp-render.js

```diff
@@ -193,6 +191,23 @@ export class AmpRender extends BaseElement {
     return super.isLayoutSupported(layout);
   }
 
+  /** @private */
+  handleResizeToContentsAction_() {
+    let currentHeight, targetHeight;
+    this.measureMutateElement(
+      () => {
+        currentHeight = this.element./*OK*/ offsetHeight;
+        targetHeight = this.element./*OK*/ scrollHeight;
+      },
+      () => {
+        if (targetHeight <= currentHeight) {
+          return;
+        }
```

Comment:
Why disallow shrinking?

Reply:
This is to short circuit since `forceChangeHeight` does not shrink because `targetHeight` is the same as the specified `height` attribute.

Reply:
Made the updates based on our offline discussion. Component can now grow or shrink height based on the content. PTAL.